tests/monitoring_test.go

```diff
@@ -287,7 +286,7 @@ var _ = Describe("Prometheus Alerts", func() {
 
 			Eventually(func(g Gomega) error {
 				return apiClient.Get(ctx, types.NamespacedName{Name: vmName, Namespace: strategy.GetNamespace()}, vm)
-			}).Should(MatchError(k8serrors.IsNotFound))
+			}).Should(MatchError(ContainSubstring(fmt.Sprintf("virtualmachines.kubevirt.io \"%s\" not found", vmName))))
```

Comment:
The default timeout for `Eventually()` is 1 second, which is probably not enough in this case.
Can you increase it? And the default polling interval is 10ms, which is too low.
https://pkg.go.dev/github.com/onsi/gomega#Eventually 

I should have commented on: https://github.com/kubevirt/ssp-operator/pull/742 , but I didn't notice it.

Reply:
@machadovilaca , please open a new PR with this change.

Reply:
https://github.com/kubevirt/ssp-operator/pull/913